microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -132,15 +132,15 @@ To enable debug tracing, add the `trace` parameter to `launch.json` , as follows
 
 Saving debug output to a log file:
 
-:::image type="content" source="./media/trace-log.png" alt-text=" Save debug output to a log file." lightbox="./media/trace-log.png":::
+![ Save debug output to a log file.](./media/trace-log.png)
       
 ```json
 ,"trace": "verbose"  // Turn on verbose tracing in the Debug Output pane.
```

Comment:
@captainbrosset - probably keep as-is.  JSON lacks support for // comments.  A workaround is add 

```
"comment" = "My comment.",
```

Reply:
I agree to keep this as is. The comment is important to the reader and needs to be somewhere. I think your workaround runs the risk of being less easy to understand. 
I believe the best solution would be to remove the comment, so that it's real valid JSON, and replace it with some text before the code snippet.
But for now, since this PR is only about images, I'd be tempted to keeping this as is.